README.md

```diff
@@ -1 +1,31 @@
-# Ruby-Enumerable-Methods
\ No newline at end of file
+# Ruby-Enumerable-Methods
+
+A program that manually performs the enumerable methods: each, each_with_index, select, all, any, none, count, inject, and map
+
+# Built With:
+Ruby
+
+# Getting Started
+ 
+ To get a copy up and running, simply run ruby enumerable_method_examples.rb
+
+
+# Authors
+
+👤 Author1
+
+GitHub: @Gracetemitope 
+Twitter: @DevGrachy 
```

Comment:
It'd give the README file a nicer look if you separate the elements each in one line by putting '-' at the beginning (for example - Github: ....